packages/engine-server/prisma/migrations/20220819014343_add_davult/migration.sql

```diff
@@ -0,0 +1,24 @@
+-- CreateTable
```

Comment:
is this path naming intentional? `packages/engine-server/prisma/migrations/20220819014343_add_davult/migration.sql`

Reply:
autogenerated by prisma